### ISSUE TITLE
addFile and fileName problems

### ISSUE THREAD

**Issue by AdrianAbba** [VeliovGroup/Meteor-Files#392]
Always if i use addFile, im getting the exception:

```
 Exception in callback of async function: ReferenceError: fileName is not defined
I20170327-17:25:45.993(2)?     at packages/ostrio_files/files.coffee:1524:58
I20170327-17:25:45.994(2)?     at wrappedCallbackForParsingMongoValidationErrors (packages/aldeed:collection2-core/collection2.js:485:15)
I20170327-17:25:45.995(2)?     at packages/mongo/collection.js:643:5
I20170327-17:25:45.995(2)?     at runWithEnvironment (packages\meteor.js:1176:24)
I20170327-17:25:45.996(2)?     at packages\meteor.js:1189:14
I20170327-17:25:45.998(2)?     at packages/mongo/mongo_driver.js:326:7
I20170327-17:25:45.999(2)?     at runWithEnvironment (packages\meteor.js:1176:24)
```

Using Windows 10
METEOR@1.4.2.7 and ostrio:files 1.7.13

**Comment by AdrianAbba** [comment]
my Code:

```
Meteor.methods({
  'excel'({ courseId}) {
    new SimpleSchema({
      courseId: String ,
    }).validate({courseId});

    if (!Meteor.userId()) {
      throw new Meteor.Error("Jesus!!!!! an Error!!!!");
    }else{
      var mongoXlsx = require('mongo-xlsx');
      var data = [];
      const userId = Meteor.userId();
      const courseProjects = Projects.find({courseId:courseId}, {supervisors:{$elemMatch:{userId: userId}}})
      courseProjects.forEach(function(project) {
        var projectObject = {name: project.courseId, id: project._id}
        data.push(projectObject);
      });
      /* Generate automatic model for processing (A static model should be used) */
      var model = mongoXlsx.buildDynamicModel(data);

      /* Generate Excel */

      if(Meteor.isServer){
        XlsFiles.denyClient();
        XlsFiles.collection.attachSchema(XlsFiles.schema);
        if(XlsFiles.findOne({userId:courseId})){
          XlsFiles.remove({userId:courseId});
        }
        mongoXlsx.mongoData2Xlsx(data, model, function(err, data) {
          console.log(data)
          console.log('File saved at:', data.fullPath);
            XlsFiles.addFile(data.fullPath, {
              name:data.fullPath,
              type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
              fileId: userId,
              userId: courseId,
              meta: {}
            });
        });
        Meteor.publish('files.xlsFiles.all', function () {
          return XlsFiles.find().cursor;
        });
      }else {
        Meteor.subscribe('files.xlsFiles.all');
      }
    }
  }
});
```

**Comment by kevinpeter** [comment]
This error occurs only in debug mode. Because the debug output is wrong. disable the debug mode to remove this error. 
The code should be: 
` console.info "[FilesCollection] [addFile]: #{opts.fileName} -> #{self.collectionName}" if self.debug` 
instead of ` console.info "[FilesCollection] [addFile]: #{fileName} -> #{self.collectionName}" if self.debug`.

This error is in file: https://github.com/VeliovGroup/Meteor-Files/blob/master/files.coffee#L1524

**Comment by dr-dimitru** [comment]
Solved in #410 , thanks to @funkyremi 
Update will be published soon.
Sorry for delay

**Comment by dr-dimitru** [comment]
@kevinpeter @AdrianAbba published as [v1.7.14](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.7.14), thank you for contribution guys.

Please, support this project by:
 - Star on [GitHub](https://github.com/VeliovGroup/Meteor-Files)
 - Star on [Atmosphere](https://atmospherejs.com/ostrio/files)
